scripts/prepare_release.sh

```diff
@@ -32,7 +32,18 @@ SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
 DATA_BROKER_ROOT=$SCRIPT_DIR/..
 
 # Update Cargo.toml versions.
-sed -i -E "s/^version = \"${VERSION_REGEX}\"$/version = \"${VERSION}\"/" \
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	if ! command -v gsed &> /dev/null; then
+    	echo "gsed could not be found; install gnu-sed"
+    	exit 1
+	else
+		SED_COMMAND="gsed" # requires gnu-sed
+	fi
+else
+	SED_COMMAND="sed"
+fi
+
+$SED_COMMAND -i -E "s/^version = \"${VERSION_REGEX}\"$/version = \"${VERSION}\"/" \
```

Comment:
on Macs preinstalled version of sed `-i` expects an argument (backup file extension) thus making the script incompatible with Mac. Since it was not easily possible to somehow extract only the different handling of the `-i flag` we are now using the compatible gsed for Mac. 
Now it should work on both Linux and Mac